src/store/selectors.ts

```diff
@@ -134,17 +134,32 @@ export const getSpread = createSelector(
     },
 );
 
+export const getSpreadInPercentage = createSelector(
+    getOpenBuyOrders,
+    getOpenSellOrders,
+    (buyOrders, sellOrders) => {
+        if (!buyOrders.length || !sellOrders.length) {
+            return new BigNumber(0);
+        }
+
+        const lowestPriceSell = sellOrders[sellOrders.length - 1].price;
+        const highestPriceBuy = buyOrders[0].price;
+        const spread = lowestPriceSell.minus(highestPriceBuy);
+        return spread.dividedBy(lowestPriceSell).multipliedBy(100);
+    },
+);
+
 export const getOrderBook = createSelector(
     getOpenSellOrders,
     getOpenBuyOrders,
     getMySizeOrders,
     getSpread,
-    (sellOrders, buyOrders, mySizeOrders, spread): OrderBook => {
+    getSpreadInPercentage,
```

Comment:
Req: These two selectors are not being used.

Reply:
I'd removed them, thanks!